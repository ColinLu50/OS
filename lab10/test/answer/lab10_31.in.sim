15425
404
315
238
313
237
