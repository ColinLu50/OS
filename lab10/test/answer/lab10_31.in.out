FCFS
75 -> 185 -> 150 -> 122 -> 81 -> 24 -> 209 -> 106 -> 51 -> 137 -> 81 -> 233 -> 16 -> 126 -> 24 -> 52 -> 13 -> 158 -> 127 -> 1 -> 113 -> 157 -> 126 -> 36 -> 33 -> 118 -> 108 -> 212 -> 77 -> 111 -> 66 -> 218 -> 101 -> 201 -> 6 -> 146 -> 92 -> 225 -> 230 -> 168 -> 185 -> 98 -> 189 -> 83 -> 37 -> 214 -> 131 -> 27 -> 237 -> 99 -> 172 -> 173 -> 182 -> 72 -> 23 -> 187 -> 133 -> 138 -> 105 -> 195 -> 73 -> 237 -> 86 -> 3 -> 123 -> 109 -> 210 -> 213 -> 30 -> 4 -> 81 -> 108 -> 158 -> 54 -> 106 -> 166 -> 240 -> 145 -> 194 -> 213 -> 122 -> 29 -> 142 -> 14 -> 13 -> 30 -> 54 -> 139 -> 190 -> 224 -> 158 -> 128 -> 94 -> 161 -> 225 -> 204 -> 70 -> 123 -> 220 -> 99 -> 212 -> 42 -> 62 -> 77 -> 37 -> 36 -> 33 -> 66 -> 73 -> 212 -> 209 -> 60 -> 71 -> 132 -> 61 -> 205 -> 86 -> 45 -> 8 -> 87 -> 117 -> 49 -> 82 -> 101 -> 184 -> 175 -> 22 -> 229 -> 81 -> 224 -> 171 -> 43 -> 24 -> 107 -> 49 -> 192 -> 189 -> 157 -> 207 -> 212 -> 46 -> 228 -> 167 -> 187 -> 137 -> 154 -> 140 -> 50 -> 96 -> 237 -> 123 -> 80 -> 19 -> 211 -> 2 -> 46 -> 20 -> 17 -> 126 -> 163 -> 136 -> 236 -> 88 -> 165 -> 188 -> 186 -> 71 -> 208 -> 160 -> 224 -> 22 -> 11 -> 179 -> 138 -> 198 -> 105 -> 240 -> 83 -> 4 -> 37 -> 120 -> 171 -> 72 -> 112 -> 22 -> 170 -> 15 -> 117 -> 218 -> 100 -> 122 -> 171 -> 98 -> 185 -> 211 -> 52 -> 104
15425
SSTF
75 -> 77 -> 77 -> 80 -> 81 -> 81 -> 81 -> 81 -> 82 -> 83 -> 83 -> 86 -> 86 -> 87 -> 88 -> 92 -> 94 -> 96 -> 98 -> 98 -> 99 -> 99 -> 100 -> 101 -> 101 -> 104 -> 105 -> 105 -> 106 -> 106 -> 107 -> 108 -> 108 -> 109 -> 111 -> 112 -> 113 -> 117 -> 117 -> 118 -> 120 -> 122 -> 122 -> 122 -> 123 -> 123 -> 123 -> 126 -> 126 -> 126 -> 127 -> 128 -> 131 -> 132 -> 133 -> 136 -> 137 -> 137 -> 138 -> 138 -> 139 -> 140 -> 142 -> 145 -> 146 -> 150 -> 154 -> 157 -> 157 -> 158 -> 158 -> 158 -> 160 -> 161 -> 163 -> 165 -> 166 -> 167 -> 168 -> 170 -> 171 -> 171 -> 171 -> 172 -> 173 -> 175 -> 179 -> 182 -> 184 -> 185 -> 185 -> 185 -> 186 -> 187 -> 187 -> 188 -> 189 -> 189 -> 190 -> 192 -> 194 -> 195 -> 198 -> 201 -> 204 -> 205 -> 207 -> 208 -> 209 -> 209 -> 210 -> 211 -> 211 -> 212 -> 212 -> 212 -> 212 -> 213 -> 213 -> 214 -> 218 -> 218 -> 220 -> 224 -> 224 -> 224 -> 225 -> 225 -> 228 -> 229 -> 230 -> 233 -> 236 -> 237 -> 237 -> 237 -> 240 -> 240 -> 73 -> 73 -> 72 -> 72 -> 71 -> 71 -> 70 -> 66 -> 66 -> 62 -> 61 -> 60 -> 54 -> 54 -> 52 -> 52 -> 51 -> 50 -> 49 -> 49 -> 46 -> 46 -> 45 -> 43 -> 42 -> 37 -> 37 -> 37 -> 36 -> 36 -> 33 -> 33 -> 30 -> 30 -> 29 -> 27 -> 24 -> 24 -> 24 -> 23 -> 22 -> 22 -> 22 -> 20 -> 19 -> 17 -> 16 -> 15 -> 14 -> 13 -> 13 -> 11 -> 8 -> 6 -> 4 -> 4 -> 3 -> 2 -> 1
404
SCAN
75 -> 73 -> 72 -> 71 -> 70 -> 66 -> 62 -> 61 -> 60 -> 54 -> 52 -> 51 -> 50 -> 49 -> 46 -> 45 -> 43 -> 42 -> 37 -> 36 -> 33 -> 30 -> 29 -> 27 -> 24 -> 23 -> 22 -> 20 -> 19 -> 17 -> 16 -> 15 -> 14 -> 13 -> 11 -> 8 -> 6 -> 4 -> 3 -> 2 -> 1 -> 0 -> 77 -> 80 -> 81 -> 82 -> 83 -> 86 -> 87 -> 88 -> 92 -> 94 -> 96 -> 98 -> 99 -> 100 -> 101 -> 104 -> 105 -> 106 -> 107 -> 108 -> 109 -> 111 -> 112 -> 113 -> 117 -> 118 -> 120 -> 122 -> 123 -> 126 -> 127 -> 128 -> 131 -> 132 -> 133 -> 136 -> 137 -> 138 -> 139 -> 140 -> 142 -> 145 -> 146 -> 150 -> 154 -> 157 -> 158 -> 160 -> 161 -> 163 -> 165 -> 166 -> 167 -> 168 -> 170 -> 171 -> 172 -> 173 -> 175 -> 179 -> 182 -> 184 -> 185 -> 186 -> 187 -> 188 -> 189 -> 190 -> 192 -> 194 -> 195 -> 198 -> 201 -> 204 -> 205 -> 207 -> 208 -> 209 -> 210 -> 211 -> 212 -> 213 -> 214 -> 218 -> 220 -> 224 -> 225 -> 228 -> 229 -> 230 -> 233 -> 236 -> 237 -> 240
315
C-SCAN
75 -> 73 -> 72 -> 71 -> 70 -> 66 -> 62 -> 61 -> 60 -> 54 -> 52 -> 51 -> 50 -> 49 -> 46 -> 45 -> 43 -> 42 -> 37 -> 36 -> 33 -> 30 -> 29 -> 27 -> 24 -> 23 -> 22 -> 20 -> 19 -> 17 -> 16 -> 15 -> 14 -> 13 -> 11 -> 8 -> 6 -> 4 -> 3 -> 2 -> 1 -> 0 -> 240 -> 240 -> 237 -> 236 -> 233 -> 230 -> 229 -> 228 -> 225 -> 224 -> 220 -> 218 -> 214 -> 213 -> 212 -> 211 -> 210 -> 209 -> 208 -> 207 -> 205 -> 204 -> 201 -> 198 -> 195 -> 194 -> 192 -> 190 -> 189 -> 188 -> 187 -> 186 -> 185 -> 184 -> 182 -> 179 -> 175 -> 173 -> 172 -> 171 -> 170 -> 168 -> 167 -> 166 -> 165 -> 163 -> 161 -> 160 -> 158 -> 157 -> 154 -> 150 -> 146 -> 145 -> 142 -> 140 -> 139 -> 138 -> 137 -> 136 -> 133 -> 132 -> 131 -> 128 -> 127 -> 126 -> 123 -> 122 -> 120 -> 118 -> 117 -> 113 -> 112 -> 111 -> 109 -> 108 -> 107 -> 106 -> 105 -> 104 -> 101 -> 100 -> 99 -> 98 -> 96 -> 94 -> 92 -> 88 -> 87 -> 86 -> 83 -> 82 -> 81 -> 80 -> 77
238
LOOK
75 -> 73 -> 72 -> 71 -> 70 -> 66 -> 62 -> 61 -> 60 -> 54 -> 52 -> 51 -> 50 -> 49 -> 46 -> 45 -> 43 -> 42 -> 37 -> 36 -> 33 -> 30 -> 29 -> 27 -> 24 -> 23 -> 22 -> 20 -> 19 -> 17 -> 16 -> 15 -> 14 -> 13 -> 11 -> 8 -> 6 -> 4 -> 3 -> 2 -> 1 -> 77 -> 80 -> 81 -> 82 -> 83 -> 86 -> 87 -> 88 -> 92 -> 94 -> 96 -> 98 -> 99 -> 100 -> 101 -> 104 -> 105 -> 106 -> 107 -> 108 -> 109 -> 111 -> 112 -> 113 -> 117 -> 118 -> 120 -> 122 -> 123 -> 126 -> 127 -> 128 -> 131 -> 132 -> 133 -> 136 -> 137 -> 138 -> 139 -> 140 -> 142 -> 145 -> 146 -> 150 -> 154 -> 157 -> 158 -> 160 -> 161 -> 163 -> 165 -> 166 -> 167 -> 168 -> 170 -> 171 -> 172 -> 173 -> 175 -> 179 -> 182 -> 184 -> 185 -> 186 -> 187 -> 188 -> 189 -> 190 -> 192 -> 194 -> 195 -> 198 -> 201 -> 204 -> 205 -> 207 -> 208 -> 209 -> 210 -> 211 -> 212 -> 213 -> 214 -> 218 -> 220 -> 224 -> 225 -> 228 -> 229 -> 230 -> 233 -> 236 -> 237 -> 240
313
C-LOOK
75 -> 73 -> 72 -> 71 -> 70 -> 66 -> 62 -> 61 -> 60 -> 54 -> 52 -> 51 -> 50 -> 49 -> 46 -> 45 -> 43 -> 42 -> 37 -> 36 -> 33 -> 30 -> 29 -> 27 -> 24 -> 23 -> 22 -> 20 -> 19 -> 17 -> 16 -> 15 -> 14 -> 13 -> 11 -> 8 -> 6 -> 4 -> 3 -> 2 -> 1 -> 240 -> 237 -> 236 -> 233 -> 230 -> 229 -> 228 -> 225 -> 224 -> 220 -> 218 -> 214 -> 213 -> 212 -> 211 -> 210 -> 209 -> 208 -> 207 -> 205 -> 204 -> 201 -> 198 -> 195 -> 194 -> 192 -> 190 -> 189 -> 188 -> 187 -> 186 -> 185 -> 184 -> 182 -> 179 -> 175 -> 173 -> 172 -> 171 -> 170 -> 168 -> 167 -> 166 -> 165 -> 163 -> 161 -> 160 -> 158 -> 157 -> 154 -> 150 -> 146 -> 145 -> 142 -> 140 -> 139 -> 138 -> 137 -> 136 -> 133 -> 132 -> 131 -> 128 -> 127 -> 126 -> 123 -> 122 -> 120 -> 118 -> 117 -> 113 -> 112 -> 111 -> 109 -> 108 -> 107 -> 106 -> 105 -> 104 -> 101 -> 100 -> 99 -> 98 -> 96 -> 94 -> 92 -> 88 -> 87 -> 86 -> 83 -> 82 -> 81 -> 80 -> 77
237
